packages/aws-rfdk/package.json

```
@@ -67,149 +67,153 @@
     "jest": true
   },
   "devDependencies": {
-    "@aws-cdk/assert": "1.116.0",
-    "@types/aws-lambda": "^8.10.81",
-    "@types/jest": "^26.0.24",
-    "@types/sinon": "^10.0.2",
-    "aws-sdk": "^2.959.0",
-    "aws-sdk-mock": "^5.2.1",
+    "@aws-cdk/assert": "1.128.0",
+    "@types/aws-lambda": "^8.10.84",
+    "@types/jest": "^27.0.2",
```

This should probably match the version of jest we're using in `cdk-build-tools` (`"^26.0.24"`) so that development type declarations match the jest library version being used to run tests.

```suggestion
    "@types/jest": "^26.0.24",
```

What do you think?

---

I updated this to match all the jest dependencies with those found in the `cdk-build-tools` .

---

Thanks!